impl/integration/env.go

```diff
@@ -167,11 +167,16 @@ func NewEnv() (*Env, error) {
 	seq := sequencer.New(logEnv.Log, mapEnv.Map, entry.New(), domainStorage, mutations, queue)
 	// Only sequence when explicitly asked with receiver.Flush()
 	d := &domaindef.Domain{
-		DomainID: domainPB.DomainId,
-		LogID:    domainPB.Log.TreeId,
-		MapID:    domainPB.Map.TreeId,
+		DomainID:    domainPB.DomainId,
```

Comment:
This code looks familiar.

Reply:
It does :-)  - will be removed in next PR.